packages/stable/src/__tests__/api/files.int.spec.ts

```diff
@@ -98,6 +113,23 @@ describe('Files integration test', () => {
     expect(aggregates[0].count).toBeDefined();
   });
 
+  test('count aggregate by geoLocation', async () => {
+    const aggregates = await client.files.aggregate({
+      filter: {
+        geoLocation: {
+          relation: 'intersects',
+          shape: {
+            type: 'Point',
+            coordinates: [10, 20],
+          },
+        },
+      },
+    });
+
+    expect(aggregates.length).toBe(1);
```

Comment:
```suggestion
    expect(aggregates.length).toBeGreaterThan(0);
```
I believe is better idea